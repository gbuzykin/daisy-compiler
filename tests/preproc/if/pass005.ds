#if true
#info "case #1"
#elif true
#info "case #2"
#elif true
#info "case #3"
#endif

#if true
#info "case #4"
#elif true
#info "case #5"
#elif false
#info "case #6"
#endif

#if true
#info "case #7"
#elif false
#info "case #8"
#elif true
#info "case #9"
#endif

#if true
#info "case #10"
#elif false
#info "case #11"
#elif false
#info "case #12"
#endif

#if false
#info "case #13"
#elif true
#info "case #14"
#elif true
#info "case #15"
#endif

#if false
#info "case #16"
#elif true
#info "case #17"
#elif false
#info "case #18"
#endif

#if false
#info "case #19"
#elif false
#info "case #20"
#elif true
#info "case #21"
#endif

#if false
#info "case #22"
#elif false
#info "case #23"
#elif false
#info "case #24"
#endif
