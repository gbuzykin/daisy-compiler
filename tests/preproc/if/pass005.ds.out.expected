./preproc/if/pass005.ds:2:2: info: "case #1"
 2 | #info "case #1"
   |  ^~~~
./preproc/if/pass005.ds:10:2: info: "case #4"
 10 | #info "case #4"
    |  ^~~~
./preproc/if/pass005.ds:18:2: info: "case #7"
 18 | #info "case #7"
    |  ^~~~
./preproc/if/pass005.ds:26:2: info: "case #10"
 26 | #info "case #10"
    |  ^~~~
./preproc/if/pass005.ds:36:2: info: "case #14"
 36 | #info "case #14"
    |  ^~~~
./preproc/if/pass005.ds:44:2: info: "case #17"
 44 | #info "case #17"
    |  ^~~~
./preproc/if/pass005.ds:54:2: info: "case #21"
 54 | #info "case #21"
    |  ^~~~
./preproc/if/pass005.ds: info: warnings 0, errors 0
